core/controllers/topic_editor.py

```diff
@@ -202,13 +204,19 @@ def get(self, topic_id):
                 summary.to_dict() for summary in skill_summaries]
             grouped_skill_summary_dicts[topic_object.name] = skill_summary_dicts
 
+        classroom_name = constants.CLASSROOM_STATUS_STAGING
+        for classroom_dict in config_domain.TOPIC_IDS_FOR_CLASSROOM_PAGES.value:
```

Comment:
Where possible, try to reuse code that is similar in multiple places. Again, you might want to have a classroom_services.py that deals with anything relating to the config property.

Reply:
No, that's not quite right -- some services do not have associated domain objects. In this case, a "classroom" is still a logical concept and should be treated as such. The only reason we don't have a domain object at the moment is because we are "cheating" by using the config property when the right way is really to be using a storage model.

So, please still create classroom_services.py and have anything that is conceptually related to "classroom" be moved there.